scripts/configure-dh.sh

```diff
@@ -0,0 +1,363 @@
+#!/bin/bash
+
+# Constants
+BACKSTAGE_CR_NAME="ai-rh-developer-hub"
+DEFAULT_RHDH_DEPLOYMENT="backstage-ai-rh-developer-hub" # deployment created by rhdh operator by default
+PLUGIN_CONFIGMAP="dynamic-plugins" # configmap created by rhdh operator for plugins by default
+EXTRA_ENV_SECRET="ai-rh-developer-hub-env" # secret created by rhdh installer to store private env vars
+
+# Variables
+BASE_DIR="$(realpath $(dirname ${BASH_SOURCE[0]}))/.."
+CATALOGS_FILE=${CATALOGS_FILE:-"${BASE_DIR}/catalogs.yaml"}
+RHDH_DEPLOYMENT="${DEFAULT_RHDH_DEPLOYMENT}"
+RHDH_EXTRA_ENV_SECRET="${EXTRA_ENV_SECRET}"
+RHDH_PLUGINS_CONFIGMAP="${PLUGIN_CONFIGMAP}"
+NAMESPACE=${NAMESPACE:-"ai-rhdh"}
+EXISTING_NAMESPACE=${EXISTING_NAMESPACE:-''}
+EXISTING_DEPLOYMENT=${EXISTING_DEPLOYMENT:-''}
+EXISTING_EXTRA_ENV_SECRET=${EXISTING_EXTRA_ENV_SECRET:-''}
+RHDH_PLUGINS=${RHDH_PLUGINS:-''}
+RHDH_INSTANCE_PROVIDED=${RHDH_INSTANCE_PROVIDED:-false}
+RHDH_GITHUB_INTEGRATION=${RHDH_GITHUB_INTEGRATION:-false}
+RHDH_GITLAB_INTEGRATION=${RHDH_GITLAB_INTEGRATION:-false}
+
+# Secret variables
+GITHUB__APP__ID=${GITHUB__APP__ID:-''}
```

Comment:
> In the case the user goes through the pipelines setup and uses the cli and enters all of the info in the prompts it should maybe set the env vars for them so they dont need to enter the info again in this script?

@Jdubrick If I'm understanding correctly this is suggesting that when a user enters this information in that we should persist the information in the env vars so that they don't need to re-enter on future runs.

This is a good idea, however for the scope of this issue and the timeline I'll want to defer it to a different task.

Reply:
Yeah sounds good to me!